### PR TITLE
Update license

### PR DESCRIPTION
**Pull request by stalmok**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2010-2016, Frédéric Hardy.
```
Comment (jubianchi) [inline LICENSE]: > every mention to "Frédéric Hardy" must be replaced with "atoum contributors"
> 
> -- #1

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Frédéric Hardy nor the names of its contributors
```
Comment (jubianchi) [inline LICENSE]: > every mention to "Frédéric Hardy" must be replaced with "atoum contributors"
> 
> -- #1

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      may be used to endorse or promote products derived from this software
+      without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY FRÉDÉRIC HARDY AND CONTRIBUTORS ``AS IS'' AND ANY
```
Comment (jubianchi) [inline LICENSE]: > every mention to "Frédéric Hardy" must be replaced with "atoum contributors"
> 
> -- #1

Here everything has tu be uppercase

File: LICENSE [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL FRÉDÉRIC HARDY AND CONTRIBUTORS BE LIABLE FOR ANY
```
Comment (jubianchi) [inline LICENSE]: > every mention to "Frédéric Hardy" must be replaced with "atoum contributors"
> 
> -- #1

Here everything has tu be uppercase

**Review by jubianchi — CHANGES_REQUESTED** [review]

**Review by jubianchi — APPROVED** [review]

**Comment by jubianchi** [comment]
Thanks!

**Comment by jubianchi** [comment]
thanks @stalmok!

**Comment by stalmok** [comment]
My pleasure!

**Pull request merged**